src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
```

Comment:
I’m not fully agree, but the codebase seems generally omit such links. So I’ll stick to Zepto code style and will remove the link.